code/modules/mob/living/carbon/human/human.dm

```diff
@@ -981,7 +981,10 @@
 				if(target.incapacitated(FALSE, TRUE) || incapacitated(FALSE, TRUE))
 					target.visible_message("<span class='warning'>[target] can't hang onto [src]!</span>")
 					return
-				buckle_mob(target, TRUE, TRUE, FALSE, 0, 2)
+				if(!isgrod(src))
+					buckle_mob(target, TRUE, TRUE, FALSE, 0, 2)
+				else
+					buckle_mob(target, TRUE, TRUE, FALSE, 0, 1)
```

Comment:
Why is this specific override needed?

Reply:
Not sure, Kapu code.
I'm going to remove it, actually.

Reply:
this enabled grod jousting